[BUG] WSL install broken

## Bug Description
Installation on WSL is not able to complete after running make install

## Environment
- **CAMI Version**: 0.4
- **OS**: WSL
- **Go Version**: 
- **Claude Code Version**: (if relevant)

## Steps to Reproduce
1. clone repo
2. run make install
3.

## Expected Behavior
Installation to continue and be smooth

## Actual Behavior
error about finding install script

## Logs/Error Messages
```
✓ Built: cami
Installing CAMI...
./install/install.sh
make: ./install/install.sh: No such file or directory
make: *** [Makefile:28: install] Error 127
```

## Additional Context
Add any other context about the problem here (screenshots, config snippets, etc.)

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,0 +1,21 @@
+# Auto detect text files and perform LF normalization
+* text=auto
+
+# Shell scripts must always use LF line endings
+*.sh text eol=lf
+
+# Makefiles must use LF
+Makefile text eol=lf
+
+# Go files
+*.go text eol=lf
+
+# Config files
+*.yaml text eol=lf
+*.yml text eol=lf
+*.json text eol=lf
+*.md text eol=lf
+
+# Windows batch files should use CRLF
+*.bat text eol=crlf
+*.cmd text eol=crlf

--- a/Makefile
+++ b/Makefile
@@ -25,7 +25,7 @@ build: ## Build CAMI binary for current platform
 
 install: build ## Build and install CAMI locally
 	@echo "Installing CAMI..."
-	./install/install.sh
+	@bash ./install/install.sh
 	@echo "✓ Installation complete!"
 
 clean: ## Remove build artifacts

